xbmc/utils/RecentlyAddedJob.cpp

```diff
@@ -258,15 +258,15 @@ bool CRecentlyAddedJob::UpdateMusic()
     home->SetProperty("LatestSong." + value + ".Fanart"  , "");
   }
   
-  i = 0;
+  size_t j = 0;
   VECALBUMS albums;
   
   if (musicdatabase.GetRecentlyAddedAlbums(albums, NUM_ITEMS))
   { 
-    for (; i < albums.size(); ++i)
+    for (; j < albums.size(); ++j)
```

Comment:
I don't get the sense here. "j" is not used after the loop, hence it makes no sense defining it outside.